design/FY2020/NFP-ZoneAirTransferModel.md

```diff
@@ -0,0 +1,143 @@
+
+This new feature provides an alternative air flow balancing method that maintains the zone mixing and/or infiltration rates at user specified flow rate.
+
+## E-mail and  Conference Call Conclusions ##
+
+NA
+
+## Overview ##
+
+This work will implement code changes that allow alternative approach of the zone air mass flow balance calculation method that does not alter the simple flow objects mass flow. 
+The fundamental zone mass balance equation remains the same but what this feature requesting is which zone air mass flow component needs to be adjusted to guarantee zone air mass flow conservation.
+
+## Approach ##
+
+Current zone air mass flow conservation equation is formulated to solve the following equation by adjusting the Zone Mixing flow, and zone infiltration flow sequentially.
+
+ 0.0 = [m_{sup} - m_{exh} - m_{ret} + m_{zmreceiving} - m_{zmsource} + m_{inf}]
+	
+where,
+m_{sup} = Zone Supply Air Mass Flow Rate, [kg/s]
+m_{exh} = Zone Exhaust Air Mass Flow Rate, [kg/s]
+m_{ret} = Zone Return Air Mass Flow Rate, [kg/s]
+m_{zmreceiving} = Zone Mixing Receiving Air Mass Flow Rate, [kg/s]
+m_{zmsource} = Zone Mixing Source Air Mass Flow Rate, [kg/s]
+m_{inf} = Zone Infiltration Air Mass Flow Rate, [kg/s]
+
+The existing procedure first attempts to solve the above equation by adjusting the Zone Mixing flow only and if that does not satisfy the above equation, then the user specified zone infiltration air flow rate is adjusted as needed.  The zone infiltration air flow can be adjusted using two different methods depending user preference. The zone infiltration air flow adjusting method is a choice input field in the ZoneAirMassFlowConservation object.
+
+The requested feature can be implemented by adjusting the return air mass flow but the return air mass flow rate can be varied between the minimum value of zero and a maximum value of the zone return air maximum flow rate. Since varying the return air mass flow only provides limiting flexibility in achieving balanced flow for the range of practical applications, the zone air flow mass balance requires two different solution scenarios: one when the zone is under positive pressure and when the zone is under negative pressure.
+
+Solving the zone air flow mass balance equation for zone return air mass flow:
+
+m_{ret} = [m_{sup} - m_{exh} + m_{zmreceiving} - m_{zmsource} + m_{inf}]
+ 
+Depending on the sign of the value of return air mass flow rate calculated using the above equation two different cases that require different solution scheme are formulated:
+
+Case I:  [m_{ret} >= 0.0]
+
+         The zone is under positive pressure and requires return air mass flow rate to balance the zone air flow. The return air flow rate will be adjusted within the bounds of 0 and maximum return air flow rate to balance the zone air mass flow.
```

Comment:
What determines "maximum return air flow rate"?
How will this work when there are multiple return nodes (more than one AHU serving a zone)?

Reply:
I am thinking the maximum will be determined based on the the Node's maximum Mass Flow Rate.

Reply:
I am thinking to use the same logic for all return nodes connected to the same zone.  Not sure if this causes any problem.

Reply:
I don't think return nodes have a defined maximum mass flow rate, but I may be wrong. The only value that might be used would be the overall airloop design flow rate, which will defeat the purpose of this feature when it wants to set return flow > supply.

Reply:
My thought is it can set zone return flow > zone supply flow but the AirLoop overall return cannot be set > the overall airloop design flow rate.